types/errors.go

```diff
@@ -245,13 +251,14 @@ func (err *sdkError) Code() CodeType {
 
 // Implements ABCIError.
 func (err *sdkError) ABCILog() string {
-	return fmt.Sprintf(`=== ABCI Log ===
-Codespace: %v
-Code:      %v
-ABCICode:  %v
-Error:     %#v
-=== /ABCI Log ===
-`, err.codespace, err.code, err.ABCICode(), err.cmnError)
+	parsedErrMsg := parseCmnError(err.cmnError.Error())
+	jsonErr := newHumanReadableError(err.codespace, err.code, err.ABCICode(), parsedErrMsg)
+	bz, er := json.Marshal(jsonErr)
```

Comment:
Can we use the Amino JSON marshal here? Keeps things more consistent.

Reply:
Let's also use must marshal json

Reply:
There's no `MustMarshalJSON` function @ValarDragon